src/scripts/Changelog.ts

```diff
@@ -136,7 +156,7 @@ const changelogItems = [
     new Changelog(changelogType.FIXED, 'Fixed breeding filter displayed options being reset after hatchery filled up'),
 
     // v0.5.1
-    new Changelog(changelogType.UPDATE, 'v0.5.1'),
+    new ChangelogUpdate('v0.5.1', new Date(2020, 7, 31)),
```

Comment:
When I backported the tags (https://github.com/pokeclicker/pokeclicker/releases), I found that the date for the commit or version bump often didn't align with the date it was released. The tags might not be any more accurate, but because some of the changelogs were retroactively updated to add missing changes, I think they might be a bit closer to the last time that version was deployed. `0.5.1` seems to have had a commit on September 4th, even though though the version bump and changelog were originally from July 31st. Not sure which ones we want to use, but I figured I'd mention it since I had the same struggle determining dates, and just ended up automating some detection for it

Reply:
Some of the versions seemed to have a patch or two afterwards, with additions to the changelog group. In these cases I went with the earlier date since that is what we would have used if we were entering dates from the start. I have made an assumption that fda8f15fb8baec3f183b5d0f39e4a1558cdeab53 and 85975e9ad7d3a5cf3d27d8b24bc7cca9d7385485 both had a release to the website though.